File: ProjectArena.Infrastructure/Models/Battle/Synchronization/SkillDto.cs
namespace ProjectArena.Infrastructure.Models.Battle.Synchronization
{
    public class SkillDto
    {
        public int Id { get; set; }

        public int Range { get; set; }

        public string NativeId { get; set; }

        public string Visualization { get; set; }

        public float Cd { get; set; }

        public int Cost { get; set; }

        public float PreparationTime { get; set; }

        public TargetsDto AvailableTargets { get; set; }

        public bool OnlyVisibleTargets { get; set; }
    }
}

File: ProjectArena.Bot/Models/Dtos/SkillDto.fs
namespace ProjectArena.Bot.Models.Dtos

type SkillDto = {
    Id: int
    Range: int
    NativeId: string
    Visualization: string
    Cd: float32
    Cost: int
    PreparationTime: float32
    AvailableTargets: TargetsDto
    OnlyVisibleTargets: bool
}